libraries/core_libs/network/src/network.cpp

```diff
@@ -128,9 +128,10 @@ void Network::sendBlock(dev::p2p::NodeID const &id, DagBlock const &blk) {
   LOG(log_dg_) << "Sent Block:" << blk.getHash().toString();
 }
 
-void Network::sendBlocks(dev::p2p::NodeID const &id, std::vector<std::shared_ptr<DagBlock>> blocks) {
+void Network::sendBlocks(dev::p2p::NodeID const &id, std::vector<std::shared_ptr<DagBlock>> blocks,
```

Comment:
hmm I see that we have a lot of chaos in all these `sendBlocks` methods :D So we have 2 options:

- either use std::vector<std::shared_ptr<DagBlock>>&&
- or const std::vector<std::shared_ptr<DagBlock>>& 

depends what is more appropriate